frontend/app/src/components/defi/Lending.vue

```diff
@@ -11,7 +11,16 @@
           :loading="allLoading"
           title="Lending"
           @refresh="refresh()"
-        />
+        >
+          <confirmable-reset
+            :loading="allLoading"
+            tooltip="Refreshes the data ignoring cached Aave historical entries"
```

Comment:
I would rewrite as:

`Refreshes the data ignoring cached Aave historical entries` -> `Refreshes the data overwriting cached Aave historical entries`